.golangci.yml

```diff
@@ -59,7 +59,7 @@ linters-settings:
       - unlambda
       - unslice
   gocyclo:
-    min-complexity: 27
+    min-complexity: 28
```

Comment:
This was required because of changes in chart.go. I'll make a separate PR to split the function in order to reduce this value again.